config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,135 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
As there is no migration pass for the 7.0 defaults, this whole file can be removed.